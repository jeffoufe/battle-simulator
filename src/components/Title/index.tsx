import styled from 'styled-components';
import { breakpoints } from '../../constants';

const Title = styled.h1`
    width: 100%;
    height: 40px;
    font-size: 1.5em;
    margin: 0;
    padding-top: 20px;
    text-align: center;
    text-transform: uppercase;

    @media only screen and (min-width: ${breakpoints.tablet}) {
        font-size: 3em;
    }
`

Title.displayName = 'Title';
export default Title;