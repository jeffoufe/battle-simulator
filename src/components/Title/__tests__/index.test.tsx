import React from 'react';
import { shallow } from 'enzyme';
import Title from '..';

describe('Title', () => {
    it('Title', () => {
        const wrapper = shallow(
            <Title>Title</Title>
        );
        expect(wrapper).toMatchSnapshot();
    })
})