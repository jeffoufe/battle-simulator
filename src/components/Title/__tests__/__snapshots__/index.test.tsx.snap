// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title Title 1`] = `
<h1
  className="sc-AxjAm iRpSIr"
>
  Title
</h1>
`;
